src/__tests__/components/CreateTemplate/SuccessNotice.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { screen, waitFor } from "@testing-library/react";
+import SuccessNotice from "@/components/CreateTemplate/SuccessNotice";
+import renderWithMockedProvider from "../../utils";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+
+describe("SuccessNotice", () => {
+  test("should render successNotice page when user punish template successfully", () => {
```

Comment:
> "it" and "test" have the same effect, but I suggest it is better to use the more semantic word, "It should" is more reasonable than "test should". Another point is to be consistent, because I found that some test files are it and some are test, which is not really good

Thank you!  have replaced "test" in all test files